### DISCUSSION TITLE
Link Figma to Storybook (production implementation)

### DISCUSSION THREAD

**Discussion by daniel-ac-martin** [UKHomeOffice/design-system#448] [Ideas]
Is there any value in linking our Storybook (https://master--60c63a8b2d881a00396fa455.chromatic.com/) to Figma?

**See:**
- https://www.chromatic.com/blog/connect-figma-to-storybook/
- https://www.chromatic.com/docs/figma

**Note:** I would need to add the Figma maintainer as a collaborator in Chromatic.

**Comment by isobelseacombe** [comment]
This could be a really good idea - will need to investigate how well the plugin functions on the free version of Figma. Once I've updated the Figma design system file maybe we can have a go with it. It would be great in the future if we had a figma file and a storybook that delivery teams could actually use rather than spinning up new ones for each product

**Reply by daniel-ac-martin** [reply]
I think that teams will often have their own Storybook, but just for their own custom components.
